tests/keras_tests/feature_networks_tests/feature_networks/weights_mixed_precision_tests.py

```diff
@@ -36,7 +36,7 @@
 
 class MixedPercisionBaseTest(BaseKerasFeatureNetworkTest):
     def __init__(self, unit_test):
-        super().__init__(unit_test)
+        super().__init__(unit_test, val_batch_size=32)
```

Comment:
Why 32? where is it coming from?

Reply:
MP num images

Reply:
But was it also 32 before in the tests or is  it something new that you're adding that would change the MP tests?